client/src/ui/Landingnavbar.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+import Image from "next/image"
+import Link from "next/link"
+import { Button, Typography } from "@mui/material";
+import "./styling/landingnavbar.css"
+import { customTheme } from "@/theme/theme";
+import {ThemeProvider} from "@mui/material";
+
+export default function Landingnavbar(){
+
+    return(
+    <nav className="navbar">
+        <div className="navbar-container">
```

Comment:
this div right here is uncessary since it's already wrapped around the nav component